doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -390,6 +392,10 @@ Wi-Fi samples
 
   * :ref:`wifi_sr_coex_sample` sample demonstrating Wi-Fi Bluetooth LE coexistence.
 
+* Changed:
+
+  * The :ref:`wifi_shell` sample now makes use of the :ref:`lib_wifi_credentials` and :ref:`wifi_mgmt_ext` libraries.
```

Comment:
```suggestion
  * The :ref:`wifi_shell_sample` sample now makes use of the :ref:`lib_wifi_credentials` and :ref:`wifi_mgmt_ext` libraries.
```
This clears the doc build failure. Dunno what to do about the conformance failure.